tests/cortexm_common_ldscript/Makefile

```diff
@@ -90,7 +90,7 @@ test-offset_%: $(BINDIR)/$(APPLICATION)_offset_%.elf
 	$(Q)echo -n "Test compilation with offset $*: "
 	$(Q)\
 	TEST_START_ADDR=$$($(PREFIX)readelf --section-headers $^ 2>/dev/null | awk '/.text/{printf "0x%s\n", $$5}'); \
-	EXPECT_START_ADDR=$$(printf "0x%08x" $$(( $(ROM_START_ADDR) + $* ))); \
+	EXPECT_START_ADDR=$$(printf "0x%08x" $$(( $(ROM_START_ADDR) + $(VTOR_OFFSET) + $(FCFIELD_OFFSET) + $* ))); \
```

Comment:
I find it strange to have here the specific `VTOR_OFFSET` and `FCFIELD_OFFSET`.

Maybe here the issue is to look for `.text` and taking only for the first `PROGBITS` section could work. (output for `tests_cortexm_common_ldscript_offset_0x1000.elf`)

For the `frdm-kw41z`

```
[ 0]                   NULL            00000000 000000 000000 00      0   0  0
[ 1] .vector           PROGBITS        00001000 001000 000400 00   A  0   0  4
```

And for `iotlab-m3`:

```
  [ 0]                   NULL            00000000 000000 000000 00      0   0  0
  [ 1] .text             PROGBITS        08001000 001000 001ef8 00  AX  0   0  4
```

If they are not needed anymore in this file, they do not need to be set in the Makefile.include at all which is somehow better I think as it is really linkerscript internal specific.

Reply:
I will split this in a separate PR.

Reply:
Ok! I was about to dot it, you can tag me to review!

Reply:
https://github.com/RIOT-OS/RIOT/pull/11588

Reply:
With the PR, `VTOR_OFFSET` and `FCFIELD_OFFSET` do not need to be set for this anymore. Currently still for `check-fcfield.sh` but it would be removed.

Reply:
`frdm-kw41z` also needs to be added to the `BOARD_WHITELIST` in this `Makefile`.